gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/exception/EndpointGroupNameAlreadyExistsException.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.v4.exception;
+
+import static java.util.Collections.singletonMap;
+
+import io.gravitee.common.http.HttpStatusCode;
+import io.gravitee.rest.api.service.exceptions.AbstractManagementException;
+import java.util.Map;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class EndpointGroupNameAlreadyExistsException extends AbstractManagementException {
+
+    private final String name;
+
+    public EndpointGroupNameAlreadyExistsException(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public int getHttpStatusCode() {
+        return HttpStatusCode.BAD_REQUEST_400;
+    }
+
+    @Override
+    public String getMessage() {
+        return "The endpoint group name [" + name + "] is already used by another endpoint or endpoint group.";
```

Comment:
not sure it's a typo but it's not consistent with the technical code
```suggestion
        return "The endpoints group name [" + name + "] is already used by another endpoint or endpoint group.";
```

Reply:
looking at the rest of the code, it seems to be singular form everywhere, so maybe we can change the technical code and parameters ? (the same apply to the other exception of the PR)

Reply:
Sorry, I'm not understand. Here it is about checking the each group name is not already used, either by another group either by an endpoint. Can you be more specific on what you think must be change?